chaco/examples/demo/basic/scatter_1d.py

```diff
@@ -69,6 +69,15 @@ def _create_plot_component():
         alignment="left",
     )
 
+    plot.plot_1d(
+        "value",
+        type="scatter_1d",
+        orientation="v",
+        marker="plus",
+        alignment="left",
+        marker_size=randint(1,5, numpts)
```

Comment:
This is not the correct usage and should fail.  You would need to specify a data source for the size values which holds the sizes.